P5-Title-Screen/content.md

```diff
@@ -298,4 +326,5 @@ The only thing left to do is get `restart` working correctly again. As it's curr
 
 Try running the game to see how it looks!
 
+<!--Take a gif of what the project looks like and put it here-->
```

Comment:
Doing this would require me to take the project apart a little, which isn't worth the time given there are others who will be in this spot in a couple days.